crates/tap-agent/src/test.rs

```diff
@@ -364,6 +378,16 @@ pub async fn store_batch_receipts(
 }
 
 pub async fn store_invalid_receipt(
+    pgpool: &PgPool,
+    signed_receipt: &TapReceipt,
+) -> anyhow::Result<u64> {
+    match signed_receipt {
+        TapReceipt::V1(signed_receipt) => store_invalid_receipt_v1(pgpool, signed_receipt).await,
+        TapReceipt::V2(_) => unimplemented!("V2 not supported"),
```

Comment:
Same thought here, why panic instead of returning an error?

Reply:
It's easier to do Search All and look for unimplemented!() calls, also these are test functions.

No receipt should be V2 if we don't parse V2 receipts in the header.